mypy/nodes.py

```diff
@@ -2559,3 +2470,215 @@ def check_arg_names(names: List[Optional[str]], nodes: List[T], fail: Callable[[
             fail("Duplicate argument '{}' in {}".format(name, description), node)
             break
         seen_names.add(name)
+
+# [Note Literals and literal_hash]
```

Comment:
Might be worth moving all the literal and literal hash specific code and comments to a new module to make `mypy.nodes` smaller. After all, these aren't relevant for most users of nodes. For example, it could be named `mypy.literals`.